Assets/Scripts/Managers/SoundManagerScript.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor.UI;
+using UnityEngine;
+
+public class SoundManagerScript : MonoBehaviour
+{
+    private static SoundManagerScript S; // this is the singleton!
+
+    public AudioClip stealthMinigameBGM; // drag clips into this field
+    public AudioSource backgroundSource;
+
```

Comment:
Add a reference for the AudioMixer and methods with float parameter to set the volume of each group.
`public AudioMixer masterMixer;`
Then to change volume you need to expose the parameters of the group and give a name. Then can call this for example to set the master volume.
`masterMixer.SetFloat("MasterVolume", newLevel)`
where "MasterVolume" is the name of exposed parameter for group and newLevel is a float parameter on method.